packages/components-library/src/components/filters/CheckboxFilter.vue

```diff
@@ -74,6 +90,11 @@ export default {
     maxVisibleOptions: {
       type: Number,
       default: () => undefined
+    },
+    showSatisfyAllCheckbox: {
+      type: Boolean,
+      required: false,
+      default: () => true
```

Comment:
Please set the default to false, else it will show up everywhere by default.

Reply:
Fixed in 1c81229